frontend/src/components/MeetingRoom/ArchivingButton/ArchivingButton.tsx

```diff
@@ -18,17 +18,20 @@ export type ArchivingButtonProps = {
  * Displays a button and handles the archiving functionality. If a meeting is currently being recorded,
  * will confirm that a user wishes to stop the recording. If a meeting is not being recorded, prompts
  * the user before starting the archive.
- * @param {ArchivingButtonProps} props - The props for the component.
- *  @property {() => void} handleClick - (optional) click handler that closes the overflow menu in small view port devices.
+ * @param {ArchivingButtonProps} props - the props for the component
+ *  @property {boolean} isOverflowButton - (optional) whether the button is in the ToolbarOverflowMenu
```

Comment:
IMO, seems like a leaky abstraction.  Even so, a tech debt item for another day 🤷‍♂️